packages/framework/src/Framework/Actions/SeedsPublicationFiles.php

```diff
@@ -0,0 +1,137 @@
+        /** @var PublicationFieldType $field */
+        foreach ($this->pubType->getFields() as $field) {
+            $lines = [];
+
+            switch ($field->type->value) {
+                case 'array':
+                    $nLines = $faker->numberBetween(3, 20);
+                    for ($i = 0; $i < $nLines; $i++) {
+                        $lines[] = $faker->word();
+                    }
+                    $output .= "$field->name:\n  - ".implode("\n  - ", $lines)."\n";
+                    $canonicalValue = $field->name == $canonicalFieldName ? $lines[0].$faker->numberBetween(1, 100000) : '';
+                    break;
+                case 'boolean':
+                    $value = $faker->numberBetween(0, 100) < 50 ? 'true' : 'false';
+                    $output .= "$field->name: $value\n";
+                    break;
+                case 'datetime':
+                    $min = $field->min ? Carbon::parse($field->min) : Carbon::today()->subDays(365)->addSeconds($faker->numberBetween(0, 86400));
+                    $max = $field->max ? Carbon::parse($field->max) : Carbon::today()->subDays(1)->addSeconds($faker->numberBetween(0, 86400));
+                    $value = Carbon::createFromTimestamp(rand($max->timestamp, $min->timestamp))->format('Y-m-d H:i:s');
+                    $output .= "$field->name: $value\n";
+                    $canonicalValue = $field->name == $canonicalFieldName ? $value : '';
+                    break;
+                case 'float':
+                    $min = $field->min ?: -10000000;
+                    $max = $field->max ?: 10000000;
+                    $value = $faker->numberBetween($min, $max) / 100;
+                    $output .= "$field->name: $value\n";
+                    $canonicalValue = $field->name == $canonicalFieldName ? $value : '';
+                    break;
+                case 'image':
+                    $min = $field->min ?: 400;
+                    $max = $field->max ?: 400;
+                    $id = $faker->numberBetween(1, 1000) / 100;
+                    $output .= "$field->name: https://picsum.photos/id/$id/$min/$max\n";
+                    break;
+                case 'integer':
+                    $min = $field->min ?: -100000;
+                    $max = $field->max ?: 100000;
+                    $value = $faker->numberBetween($min, $max);
+                    $output .= "$field->name: $value\n";
+                    $canonicalValue = $field->name == $canonicalFieldName ? $value : '';
+                    break;
+                case 'string':
+                    $min = $field->min ?: 0;
+                    $max = $field->max ?: 255;
+                    $value = Str::of($faker->sentence(10))->limit($faker->numberBetween($min, $max), '...');
+                    $output .= "$field->name: $value\n";
+                    $canonicalValue = $field->name == $canonicalFieldName ? $value : '';
+                    break;
+                case 'tag':
```

Comment:
I'm still not able to get the tags to work, could you explain what they're supposed to do @rgasch?

Reply:
I will take a look tonight ...